drivers/power_domain/Kconfig

```diff
@@ -12,6 +12,12 @@ module = POWER_DOMAIN
 module-str = power_domain
 source "subsys/logging/Kconfig.template.log_config"
 
+config POWER_DOMAIN_INIT_PRIORITY
```

Comment:
This kconfig should be applied in the other power domain driver

Reply:
Why do we will have one priority for all power domains ? Different power domains can have different priorities, I wouldn't have it.

Reply:
You want to have a kconfig symbol to this domain, please create one specific for this domain ... `POWER_DOMAIN_GPIO_PRIORITY_INIT_PRIORITY`. It does not make sense having one priority for all domains  (even if now they are using the same value).

Reply:
Was this discussion resolved before merge? Afraid not.